sync-core/src/main/java/com/cloudant/sync/query/Index.java

```diff
@@ -177,6 +176,25 @@ protected boolean compareIndexTypeTo(String indexType, String indexSettings) {
         return this.indexSettings.equals(settings);
     }
 
+    /**
+     * Converts the index settings to a JSON string
+     *
+     * @return the JSON representation of the index settings
+     */
+    protected String getIndexSettingsAsAString() {
```

Comment:
I think this should be named something like `toJSONString` it makes it clearer from the method name the kind of string that is being returned